templates/travel/partials/icon/_icon.css

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2016 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.travel-icon, [class^="travel-img-icon"] {
+  display: inline-block;
+  vertical-align: -23%;
+  height: var(--icon-size);
+  width: var(--icon-size);
+
+  background-position: 50% 50%;
+  background-repeat: no-repeat;
+  background-size: auto var(--icon-size);
+}
+
+.travel-icon:not(:root) {
```

Comment:
why do you need the :not?

Reply:
```svg:not(:root)``` is a user agent stylesheet attr in chrome and firefox, but not IE.